recipe/meta.yaml

```diff
@@ -102,10 +102,22 @@ outputs:
       run:
         - {{ pin_compatible("cuda-version", max_pin="x.x") }}
         - {{ pin_subpackage("libcublas", exact=True) }}
+        - cuda-cudart-dev_{{ target_platform }}
+        - cuda-crt-dev_{{ target_platform }}
```

Comment:
Was this just meant to be the CRT headers?

```suggestion
        - cuda-crt-dev_{{ target_platform }}
```

Reply:
We thought all the headers would be found in CRT feedstock alone but it turns out cuda_runtime.h comes from cuda-cudart-dev